libcodechecker/libhandlers/analyze.py

```diff
@@ -137,6 +137,16 @@ def add_arguments_to_parser(parser):
                         help="Annotate the ran analysis with a custom name in "
                              "the created metadata file.")
 
+    parser.add_argument('-f', '--force',
```

Comment:
I suggest to change the parameter name to 'clean' from 'force'. We usually use the force parameter to enforce some potentially dangerous action like (rm -rf) which otherwise would trigger interactive questions from users. If this parameter is omitted, then there is no such question, but the analysis runs in incremental mode. 

So the clean mode expresses the fact that we clean the previous results and perform a new analysis.